.azure-pipelines/client.yml

```diff
@@ -0,0 +1,180 @@
+trigger:
+- psSdkJson6
+
+#Variables: 
+#  'source.scope' list of projects to build (Ex: SDKs\KeyVault\dataPlane)
+#  'build.useLatest' indicates whether or not to use build bootstrap files from the LTSBuildTools branch
+#  'sign.pattern' the pattern used to find assemblies for SN and AuthentiCode signing
+#  'system.debug' bool for debug tracing
+
+#TODO: sign.pattern needs better filtering (not KeyVault specific, exclude test assemblies).
+
+variables:
+  source.scope: 'SDKs\KeyVault\dataPlane'
+  build.useLatest: 'false'
+  sign.pattern: '*KeyVault*.dll'
```

Comment:
Why do we need this pattern? Can it not just be `*.dll`?

Reply:
You and I are working on this.

Reply:
We've decided to remove signing from this yml and try to accomplish it with the signtool in the release pipeline.